app/src/main/java/vunt/com/vn/moviedb_28/data/model/Movie.java

```diff
@@ -177,7 +177,7 @@ public Movie setSpokenLanguages(List<Language> spokenLanguages) {
     }
 
     public float getVoteAverage() {
-        return mVoteAverage;
+        return mVoteAverage/2 ;
```

Comment:
các hàm get và set thông thường để lấy ra giá trị của field 
Em nên đặt tên khác 
calculateAbcxyz....

Reply:
nhớ format code em nhé 
```suggestion
        return mVoteAverage / 2;
```